src/nominatim_db/tools/special_phrases/sp_importer.py

```diff
@@ -200,13 +203,14 @@ def _create_classtype_table_and_indexes(self,
         with self.db_connection.cursor() as db_cursor:
             db_cursor.execute("CREATE INDEX idx_placex_classtype ON placex (class, type)")
 
-        allowed_special_phrases = self.get_classtype_pairs()
+        allowed_special_phrases = self.get_classtype_pairs(min)
```

Comment:
When min is 0, then you don't even need to run this function. Saves us a database lookup.